### DISCUSSION TITLE
I tried to highlight the outline of the object, but the parameter configuration did not work. Can you give me some guidance? Thank you

### DISCUSSION THREAD

**Discussion by Haidiye00** [google/filament#7179] [Q&A]
these are my test code: 
![8aba60de-b1ef-4711-89f5-beff77fbef4c](https://github.com/google/filament/assets/132528822/15fbbf49-ffdf-41f6-a0c3-09edc86bfe46)

This is the effect I expected  ：show the highlighted contour when  the object is selected：
![4f1a7656-7d81-4bb2-a03e-a9c77c07a5d5](https://github.com/google/filament/assets/132528822/521c6663-1eb5-451e-90ac-2f4837b73ea2)

**Comment by bejado** [comment]
You also need to ensure you [enable the stencil buffer](https://github.com/google/filament/blob/9474798c753ae39b0ba93c3d2e8f5575eb103440/filament/include/filament/View.h#L672) on the `View`.

**Reply by Haidiye00** [reply]
thanks
